### PR TITLE
[ChargeItem] Expose Priority on ChargeItem.Reason

### PR DESCRIPTION
**Pull request by Coffeevee**
Description
----
Update to ChargeItem.Reason to expose the Priority field.
Update Priority to be a standalone extension.

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

Screenshots
----
Screen Shot 2021-05-12 at 1 05 17 PM
Screen Shot 2021-05-12 at 1 07 09 PM
Screen Shot 2021-05-12 at 1 07 23 PM
Screen Shot 2021-05-12 at 1 07 55 PM

### CODE REVIEW

File: content/millennium/r4/conformance/structure-definition.md [hunk 1]
```diff
@@ -151,6 +152,7 @@ Authorization is not required.
 [`performing-location`]: https://fhir-ehr.cerner.com/r4/StructureDefinition/performing-location?_format=json
 [`period`]: https://fhir-ehr.cerner.com/r4/StructureDefinition/period?_format=json
 [`pharmacy-verification-status`]: https://fhir-ehr.cerner.com/r4/StructureDefinition/pharmacy-verification-status
```
Comment (Coffeevee) [inline content/millennium/r4/conformance/structure-definition.md:154]: Do we want to update this URL to include `?_format=json`?

File: content/millennium/r4/conformance/structure-definition.md [hunk 2]
```diff
@@ -106,6 +106,7 @@ Authorization is not required.
  [`performing-location`]                                     | A location where the resource was performed.
  [`period`]                                                  | A time period defined by a start and end date/time.
  [`pharmacy-verification-status`]                            | Represents whether a MedicationRequest has been verified by a pharmacist.
+ [`priority`]                                                | The priority of the field or element that is a list.
```
Comment (alexbeckner1) [inline content/millennium/r4/conformance/structure-definition.md:109]: I thought the definition that we landed was "The priority of the element within a list."

Comment (Coffeevee) [reply content/millennium/r4/conformance/structure-definition.md]: Updated - https://github.com/cerner/fhir.cerner.com/pull/639/commits/f39ebcb7f8d1d494cd811d7f0eaee9367b87b8e2#diff-f10dc1eb3735aa1264e82ae17b8a0cd967fc69912568786cd6406be319d3fd17R109

File: content/millennium/r4/conformance/structure-definition.md (hunk 1 above)
Comment (exopsRamba) [reply content/millennium/r4/conformance/structure-definition.md:154]: Nah, it's not something we've ever touched so let's leave it.

**Comment by exopsRamba** [comment]
> Our current example for ChargeItem read doesn't include a Reason field. Do we need to include one?

It would be nice if we could find an example with all of our fields, but it's not always possible.

**Comment by exopsRamba** [comment]
Screenshots need to be updated.

**Comment by exopsRamba** [comment]
I think we should add priority to this list now that it isn't directly tied to an extension's sub fields. https://github.com/cerner/fhir.cerner.com/blame/main/content/millennium/r4/financial/charge-item.md#L52

cc. @alexbeckner1

**Comment by Coffeevee** [comment]
> > Our current example for ChargeItem read doesn't include a Reason field. Do we need to include one?
> 
> It would be nice if we could find an example with all of our fields, but it's not always possible.

I agree that it would be a quality-of-life thing to have, but since priority appears as a nested extension in other places, I think I'll just leave the example as-is unless you have strong feelings about it.

**Comment by alexbeckner1** [comment]
> I think we should add priority to this list now that it isn't directly tied to an extension's sub fields. https://github.com/cerner/fhir.cerner.com/blame/main/content/millennium/r4/financial/charge-item.md#L52
> 
> cc. @alexbeckner1

+1 please add

**Comment by Coffeevee** [comment]
> Screenshots need to be updated.

Screenshots are now updated. Thanks for the reminder :)

**Comment by Coffeevee** [comment]
> > I think we should add priority to this list now that it isn't directly tied to an extension's sub fields. https://github.com/cerner/fhir.cerner.com/blame/main/content/millennium/r4/financial/charge-item.md#L52
> > cc. @alexbeckner1
> 
> +1 please add

Added: https://github.com/cerner/fhir.cerner.com/pull/639/commits/a826d3bb199660afd64f40946f2eaf3a2b619216#diff-16d4f8f74626c62d57a8b71d6be699f60e72fffed8636bc32e6654d4252b3ebbR66

**Review by exopsRamba — APPROVED** [review]

**Review by Anurag-Balannagari — APPROVED** [review]

**Review by ahamdany — APPROVED** [review]

**Review by MadhurRaj — APPROVED** [review]

**Comment by MadhurRaj** [comment]
Screen Shot 2021-05-22 at 6 59 24 PM
Screen Shot 2021-05-22 at 6 59 06 PM
Screen Shot 2021-05-22 at 6 58 51 PM

**Pull request merged**